src/manifolds/Sphere.jl

```diff
@@ -365,7 +366,7 @@ norm for the case $m>1$.
 """
 project(::AbstractSphere, ::Any)
 
-project!(S::AbstractSphere, q, p) = copyto!(q, p ./ norm(p))
+project!(::AbstractSphere, q, p) = (q .= p ./ norm(p))
```

Comment:
I though we wanted to switch to `copyto!`? Or is that only if we speak about identity operations?

Reply:
Yeah just for identity. If we use `copyto!` here, it will allocate and then copy, whereas this is non-allocating.

Reply:
Cool, thanks for the explanation. As I already mentioned, I will migrate code when I see it.